src/components/Showcase.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState } from "react"
+import styled from "styled-components"
+import {
+  Card,
+  CardActionArea,
+  Divider,
+  MuiThemeProvider,
+  SvgIcon,
+  Typography,
+} from "@material-ui/core"
+import Grid from "@material-ui/core/Grid/Grid"
+import { bps, textThemeDark } from "theme/index"
+import { AutoGrowSpacer, VerticalSpacer } from "components/Spacer"
+import TuneRoundedIcon from "@material-ui/icons/TuneRounded"
+import SnoozeRoundedIcon from "@material-ui/icons/SnoozeRounded"
+import CreditCardRoundedIcon from "@material-ui/icons/CreditCardRounded"
+import CustomResourceVid from "resources/video/custom_resource.gif"
+import BillingVid from "resources/video/billing.gif"
+import SleepModeVid from "resources/video/sleep_mode.gif"
+import ActionButton from "components/Button"
+import PowerSettingsNewRoundedIcon from "@material-ui/icons/PowerSettingsNewRounded"
+
+const StyledVideoTabContainer = styled.div`
+  background-color: ${props => props.theme.palette.background.paper};
+
+  .disabledIcon {
+    width: 40px;
+    height: 40px;
+    color: ${props => props.theme.palette.text.disabled};
+  }
+
+  .activeIcon {
+    width: 40px;
+    height: 40px;
+    color: ${props => props.theme.palette.primary.main};
+  }
+
+  .videoTextList {
+    max-width: 400px;
+  }
+
+  .pricing-btn {
+    margin-left: 20px;
+  }
+
+  .videoSrc {
+    width: auto;
+    height: 450px;
+    border-radius: 8px;
+    box-shadow: 0px 4px 30px 0px rgba(0, 0, 0, 0.14),
+      0px 1px 10px 0px rgba(0, 0, 0, 0.12), 0px 2px 4px -1px rgba(0, 0, 0, 0.2);
+
+    ${bps.down("md")} {
+      padding: 0px 16px;
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+      overflow: hidden;
+    }
+  }
+`
+
+const VideoTab = () => {
+  const tabData = [
+    {
+      id: "1",
+      name: "Custom Resources",
+      info:
+        "Scale down to a single shared 32 Mb instance or up multi-instance, multi-CPU workloads.",
+      icon: <TuneRoundedIcon />,
+      src: CustomResourceVid,
+    },
+    {
+      id: "2",
+      name: "Sleep Mode",
+      info:
+        "Experience true cost savings with dev environments that automatically sleep when they are not in use!",
+      icon: <SnoozeRoundedIcon />,
+      src: SleepModeVid,
+    },
+    {
+      id: "3",
+      name: "Transparent Billing",
+      info: "Understand your maximum monthly cost before deploying anything!",
+      icon: <CreditCardRoundedIcon />,
+      src: BillingVid,
+    },
+  ]
```

Comment:
you can put this outside the functional component as a constant
so every time the component re-renders this will not be created again